src/training.rs

```diff
@@ -174,7 +174,7 @@ impl DashboardRenderer for ProgressCollector {
 pub(crate) struct TrainingConfig {
     pub model: ModelConfig,
     pub optimizer: AdamConfig,
-    #[config(default = 16)]
+    #[config(default = 216)]
```

Comment:
Was this deliberate? With such a big jump in epochs, we'll lose any performance gains we've made here.